packages/formstr-app/src/containers/CreateFormNew/components/AIFormGeneratorModal/OllamaSettings.tsx

```diff
@@ -0,0 +1,100 @@
+import { QuestionCircleOutlined } from '@ant-design/icons';
+import { OllamaSettingsProps } from './types';
+
+const { Text, Paragraph, Link, Title } = Typography;
+const { Panel } = Collapse;
+const CodeBlock: React.FC<{ children: React.ReactNode }> = ({ children }) => (
+  <pre style={{ background: '#f5f5f5', padding: '10px', borderRadius: '4px', overflowX: 'auto', marginTop: '8px', marginBottom: '8px' }}>
+    <code>{children}</code>
+  </pre>
+);
+
+const OllamaSettings: React.FC<OllamaSettingsProps> = ({
+  ollamaUrl,
+  onUrlChange,
+  onTestConnection,
+  loading,
+}) => {
+  return (
+    <Space direction="vertical" size="small" style={{ width: '100%' }}>
+      <div>
+        <Space align="center" style={{ marginBottom: 4 }}>
+          <Text strong>Ollama Server URL</Text>
+          <Tooltip title="The base URL of your running Ollama instance (e.g., http://localhost:11434). Connection will be tested automatically when you click away/blur from this field.">
+            <QuestionCircleOutlined style={{ color: 'rgba(0, 0, 0, 0.45)', cursor: 'help' }} />
+          </Tooltip>
+        </Space>
+        <Input
+          placeholder="e.g., http://localhost:11434"
+          value={ollamaUrl}
+          onChange={onUrlChange}
+          onBlur={onTestConnection}
+          disabled={loading}
+          aria-label="Ollama Server URL"
+        />
+      </div>
+
+      <Collapse ghost style={{ width: '100%', margin: '0' }}>
+        <Panel header={<Text type="secondary">Show Ollama Setup Steps</Text>} key="1">
+          <Paragraph>
+            To use the AI form generation feature with your local Ollama instance, follow the steps below for your operating system.
+          </Paragraph>
+          <AntDivider />
+
+          <Title level={5}>1. Download Ollama & Pull Model (All Platforms)</Title>
+          <Paragraph>
+            Download and install Ollama from the official site:
+            <br />
+            <Link href="https://ollama.com/download" target="_blank" rel="noopener noreferrer">
+              https://ollama.com/download
+            </Link>
+          </Paragraph>
+          <Paragraph>
+            After installation, open your terminal (macOS/Linux/WSL/Windows PowerShell) and pull a model:
+          </Paragraph>
+          <CodeBlock>ollama pull llama3</CodeBlock>
+          <Paragraph type="secondary">(e.g., <Text code>llama3</Text>, or your preferred model)</Paragraph>
+          <AntDivider />
+          <Title level={5}>2. Start Ollama Server (macOS / Linux / Windows)</Title>
+          <Paragraph>
+            Use the command block below based on your platform to start the Ollama server:
+          </Paragraph>
+
+          <Row gutter={16}>
+            <Col xs={24} md={12}>
+              <Text strong>macOS / Linux / WSL</Text>
+              <CodeBlock>
+                export OLLAMA_HOST=0.0.0.0:11434{'\n'}
+                export OLLAMA_ORIGINS='*'{'\n'}
```

Comment:
a better approach would be to add the `host (formstr.app)`` to ollama origins instead of allowing everything (*) also on most machines ollama starts automatically so we should give instructions on restart after setting variables maybe? also for persistent effect on linux i had to edit `/etc/systemd/system/ollama.service` for it to take effect.